### PR TITLE
upgrade to .Net6.0 & fix bug for sample GitHubChat

### PR DESCRIPTION
**Pull request by xingsy97**
1\ upgrade to .Net 6.0 and still works. 
2\ there is a small bug which caused error when `user["company"]` = `null`

### CODE REVIEW

**Comment by dnfgituser** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/aspnet/AzureSignalR-samples?pullRequest=182) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/aspnet/AzureSignalR-samples?pullRequest=182) before we can accept your contribution.<br/><br/>:x: xingsy97 [sign now](https://cla.dotnetfoundation.org/aspnet/AzureSignalR-samples?pullRequest=182)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/aspnet/AzureSignalR-samples?pullRequest=182) it.</sub>

File: samples/GitHubChat/Startup.cs [hunk 1]
```diff
@@ -80,7 +80,7 @@ private static async Task GetUserCompanyInfoAsync(OAuthCreatingTicketContext con
                 HttpCompletionOption.ResponseHeadersRead, context.HttpContext.RequestAborted);
 
             var user = JsonSerializer.Deserialize<Dictionary<string, object>>(await response.Content.ReadAsStringAsync());
-            if (user.ContainsKey("company"))
+            if (user.ContainsKey("company") && user["company"] != null)
```
Comment (vicancy) [inline samples/GitHubChat/Startup.cs:83]: use `user.TryGetValue("company", out var ...)`

Comment (xingsy97) [reply samples/GitHubChat/Startup.cs:83]: - `dict.TryGetValue(key, out value)` only checks whether `dict` contains `keys` without checking its value

- Actually `"company"` is indeed a key in dict `user` and `user["company"] = null`

- That means `user.TryGetValue("company", out company)` is always `=true` and our code will not work.

- So `TryGetValue` doesn't fit this situation.

Comment (vicancy) [reply samples/GitHubChat/Startup.cs:83]: Use `dict.TryGetValue(key, out var value) && value != null` to avoid fetching the dict twice

Comment (xingsy97) [reply samples/GitHubChat/Startup.cs]: updated

File: samples/GitHubChat/Startup.cs [hunk 2]
```diff
@@ -80,12 +80,12 @@ private static async Task GetUserCompanyInfoAsync(OAuthCreatingTicketContext con
                 HttpCompletionOption.ResponseHeadersRead, context.HttpContext.RequestAborted);
 
             var user = JsonSerializer.Deserialize<Dictionary<string, object>>(await response.Content.ReadAsStringAsync());
-            if (user.ContainsKey("company") && user["company"] != null)
+            object company;
+            if (user.TryGetValue("company", out company) && company != null)
```
Comment (vicancy) [inline samples/GitHubChat/Startup.cs:84]:
```suggestion
            if (user.TryGetValue("company", out var company) && company != null)
```

**Review by vicancy — APPROVED** [review]

File: samples/GitHubChat/Startup.cs (hunk 2 above)
Comment (xingsy97) [reply samples/GitHubChat/Startup.cs]: updated

**Review by vicancy — APPROVED** [review]

**Pull request merged**